README.md

```diff
@@ -1,25 +1,31 @@
 # preCICE adapter for MBDyn #
 
 ## Installation ##
+No installation configured, instead include folder in PYTHONPATH
 ```
-sudo python setup.py install
+export PYTHONPATH=$PYTHONPATH:<path-to-adapter>
 ```
 
 Dependencies:
-*    MBDyn with Python interface
-*    preCICE with Python interface
+*    MBDyn with Python interface, develop branch for python 3 support
+*    preCICE with Python interface versio 2 or higher
```

Comment:
The appropriate instruction would be:
* preCICE <version>
* preCICE python-bindings <version>